contract/src/contract.js

```diff
@@ -48,7 +48,10 @@ export const makeContract = harden(zcf => {
       rejectOffer(offerHandle, `We are no longer giving encouragement`);
     }
 
+    const { instanceHandle } = zcf.getOffer(offerHandle);
     const userTipAllocation = zcf.getCurrentAllocation(offerHandle).Tip;
+    const { brandKeywordRecord } = zcf.getInstanceRecord(instanceHandle);
```

Comment:
`zcf.getInstanceRecord` already knows the instanceHandle, so it doesn't take an argument. I think we can combine all of this into one line:

```suggestion
    const tipAmountMath = zcf.getAmountMath(zcf.getInstanceRecord().brandKeywordRecord.Tip);
```

Reply:
Thanks. I'll drop the param.

Since we need the brandKeywordRecord both here and to get the adminTip, I'll leave the rest of it as is.